java/spacewalk-java.changes

```diff
@@ -1,3 +1,4 @@
+- Speed up getAllChannels api call (bsc#1153234)
```

Comment:
While you are at it, please also change the wording to highlight the change from a user perspective, eg. "improve speed of Content Lifecycle Management channel list loading"

Reply:
this has already merged in 4.0